src/components/AssetSpecs/AddAssetSpecComponent.jsx

```diff
@@ -24,7 +24,7 @@ const AddAssetSpecComponent = (props) => {
   return (
     <Form onSubmit={props.handleSubmit}>
       <Form.Field>
-        <label htmlFor="year_of_manufacture" className="label-style">Year of Manufacture</label>
+        <div className="label-style">Year Of Manufacture</div>
```

Comment:
Why the change from `label` to `div`? `label` usually helps with accessibility. Please read more about [semantic HTML](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML) and their role in accessibility

Reply:
This is how the previous forms have been implemented. I think we should have a chore to change the rest as well.